examples/linear_model/plot_logistic_multinomial_rcv1.py

```diff
@@ -0,0 +1,101 @@
+"""
```

Comment:
The filename is misleading, it should be `plot_sparse_logistic_regression_20newsgroups.py`:

- 20newsgroups instead of RCV1,
- highlight the use of l1 penalty.